### ISSUE TITLE
Make "Back to previous page" link more visible on form pages

### ISSUE THREAD

**Issue by cindymerrill** [department-of-veterans-affairs/va.gov-team#136902]
## Background
- Study: 2026-01 Save in Progress (2nd round of usability testing)
- Date: 2026-03-17
- Team: ADE (Accessibility Digital Experience)
- [Full Research Findings Document](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/digital-experience/ADE/research/2026-01-save-in-progress-2/research-findings.md)

## Research Insight
- Recommendation: Make the "Back to previous page" link easier to notice. The current font size (16px) is too small, which should be enlarged to 16.96 px.
- Supporting evidence: One non-AT participant said they did not notice this link at the top of the page due to its small size.

## Proposed Action
- Increase the font size of the "Back to previous page" link to 16.96px 
- Consider adding a second "Back to previous page" link at the bottom of form pages

## Acceptance Criteria
- [ ] "Back to previous page" link font size is increased to recommended size
- [ ] Secondary link is added at page bottom (if decided appropriate)

## Suggested Labels
- research
- accessibility
- usability
- enhancement
- priority-medium
- ADE
- ADE Research
- ADE Internal Research

## Priority Ranking
- Medium (Improves navigation and user satisfaction)

## Related Issues or Epics
- May be grouped with other form usability improvements
